### DISCUSSION TITLE
How to configure multiple languages ?

### DISCUSSION THREAD

**Discussion by AtlantisDe** [mkdocs/mkdocs#2723] [Q&A]
### How to configure multiple languages

- I didn't find the information, thank you

**Comment by Andre601** [comment]
For once there is [this discussion](https://github.com/squidfunk/mkdocs-material/discussions/2346) on how you can do it manually.

Then there is also the [`static i18n` plugin](https://github.com/ultrabug/mkdocs-static-i18n) that does most of the heavy lifting and leaves you with just making files to use.

If you need more help, try explaining what your general goal here is and what your setup would be.
